src/main/java/com/fasttime/domain/member/service/MemberService.java

```diff
@@ -1,66 +1,68 @@
 package com.fasttime.domain.member.service;
 
+
+import com.fasttime.domain.member.repository.FcMemberRepository;
+
+import java.time.LocalDateTime;
 import org.springframework.stereotype.Service;
 
 import com.fasttime.domain.member.entity.Member;
 import com.fasttime.domain.member.dto.MemberDto;
 import com.fasttime.domain.member.repository.MemberRepository;
-import com.fasttime.domain.member.exception.UserNotFoundException;
+
 
 import lombok.RequiredArgsConstructor;
 
 
 import org.springframework.transaction.annotation.Transactional;
-import java.util.Optional;
+
 
 @Service
 @Transactional
 @RequiredArgsConstructor
 public class MemberService {
 
     private final MemberRepository memberRepository;
-
+    private final FcMemberRepository fcMemberRepository;
 
 
     public void save(MemberDto memberDto) {
-
+        //패스워드 인코딩(security적용 후 주석 해제)
+        //String encodedPassword = passwordEncoder.encode(memberDto.getPassword());
 
         Member member = new Member();
         member.setEmail(memberDto.getEmail());
-        member.setPassword(memberDto.getPassword()); // 인코딩된 패스워드 저장?
+        member.setPassword(memberDto.getPassword());
+        //security적용 후 주석 해제
+        //member.setPassword(encodedPassword);
         member.setNickname(memberDto.getNickname());
 
         memberRepository.save(member);
     }
 
-
-    public void save(Member member) {
-        memberRepository.save(member);
+    public boolean isEmailExistsInFcmember(String email) {
+        return fcMemberRepository.existsByEmail(email);
     }
 
-/*
+    public boolean isEmailExistsInMember(String email) {
+        return memberRepository.existsByEmail(email);
+    }
 
-    public Member findByEmail(String email){
-        return fc_memberRepository.findByEmail(email)
 
-            .orElseThrow(() -> new UserNotFoundException("유저를 찾을 수 없습니다." + email));
+    public void save(Member member) {
+        memberRepository.save(member);
     }
-*/
-
 
 
-    public boolean checkDuplicateNickname(String username) {
-        return memberRepository.findByNickname(username).isPresent();
+    public boolean checkDuplicateNickname(String nickname) {
+        return memberRepository.findByNickname(nickname).isPresent();
     }
 
 
-
-    public void deleteMember(int userId) throws UserNotFoundException {
-        Optional<Member> member = memberRepository.findById(userId);
-        if (!member.isPresent()) {
-            throw new UserNotFoundException("User not found with id: " + userId);
-        }
-        memberRepository.delete(member.get());
+    public void softDeleteMember(Member member) {
+        //deleted_at 칼럼에 값이 있으면 탈퇴된 회원으로 간주
+        member.setDeletedAt(LocalDateTime.now()); // 현재 시간으로 소프트 삭제 시간 설정
```

Comment:
`nit` : `deletedAt()` 네이밍은 다소 애매해보일 수 있습니다. 확실하게 삭제의 느낌을 주는건 어떨까요?
```suggestion
        member.delete(LocalDateTime.now());
```

Reply:
> `nit` : `deletedAt()` 네이밍은 다소 애매해보일 수 있습니다. 확실하게 삭제의 느낌을 주는건 어떨까요?

동의합니다. 아무래도 제가 이해하기 편한 방향으로 코드를 작성하다 보니 이런 네이밍이 나온 것 같습니다 ㅎㅎ 수정하겠습니다!